content/_data/upgrades/2-235-1.adoc

```diff
@@ -0,0 +1,23 @@
+==== Plugin manager improvements
+
+The plugin manager interface includes significant improvements.
+The "Available" tab previously showed all available plugins then filtered based on the value in the "search" field.
+The "Available" tab now has improved search facilities and only shows plugins that match the value in the "search" field.
+Users who previously scrolled through over 1000 plugins to find their plugin will now need to enter a value in the search field.
+
+Plugin updates that resolve security issues are shown in the "Updates" tab with a 'Security' label.
```

Comment:
Not true, there's a block below the description similar to compatibility issues.

Reply:
Section has been removed from the upgrade guide.  As you've noted in [your comment](https://github.com/jenkins-infra/jenkins.io/pull/3451#discussion_r439935778), upgrade guide is intended for specific information related to upgrades, not general feature descriptions.

I downgraded the script security plugin on my installation and restarted, and I saw a label 'Security' which appeared but did not see anything else.  That's what motivated me to describe it this way originally.  No longer an issue since the text has been removed.